groups/chips-rep-db/profiles/heritage-staging-eu-west-2/vars

```diff
@@ -108,7 +108,8 @@ chips_rep_db_sg = [
   "sgr-chips-read-only-asg-001-*",
   "sgr-chips-db-batch-asg-001-*",
   "sgr-windows-workloads-bus-obj-2-server-*",
-  "sgr-chips-oltp-logstb-db-ec2-001-*"
+  "sgr-chips-oltp-logstb-db-ec2-001-*",
+  "sgr-chips-oem-ec2-001-*"
 ]
```

Comment:
See https://github.com/companieshouse/chips-terraform/pull/230/files#r1210214928

It may be better to define this in a separate list so it can be targeted specifically for OEM port access.